python/ray/rllib/tuned_examples/regression_tests/pendulum-ddpg.yaml

```diff
@@ -0,0 +1,15 @@
+pendulum-ddpg:
+    env: Pendulum-v0
+    run: DDPG
```

Comment:
DDPG2 here and elsewhere in the YAML examples.

Btw, how long does this usually take to complete?

Reply:
Done. It takes around 650 to 750 seconds.